holoviews/plotting/renderer.py

```diff
@@ -257,3 +284,10 @@ def state(cls):
         """
         yield
 
+
+    @classmethod
+    def validate(cls, options):
+        """
+        Validates a dictionary of options set on the backend.
```

Comment:
Also, why not make ``validate`` accept ``**kwargs`` instead of a dictionary of options? Ends up being the same thing but maybe a nicer interface for validation. Just something to consider.